lib/commands/file-movement.js

```diff
@@ -47,12 +48,19 @@ async function parseContainerPath (remotePath, containerRootSupplier) {
     log.errorAndThrow(`It is expected that package identifier is separated from the relative path with a single slash. ` +
                       `'${remotePath}' is given instead`);
   }
-  const containerRoot = _.isFunction(containerRootSupplier) ?
-    (await containerRootSupplier(match[1])) :
-    containerRootSupplier;
+  let bundleId = match[1];
+  let containerType = null;
+  const typeSeparatorPos = bundleId.indexOf(CONTAINER_TYPE_SEPARATOR);
+  if (typeSeparatorPos > 0 && typeSeparatorPos < bundleId.length - 1) {
+    bundleId = bundleId.substring(0, typeSeparatorPos);
+    containerType = bundleId.substring(typeSeparatorPos + 1);
+  }
+  const containerRoot = _.isFunction(containerRootSupplier)
+    ? await containerRootSupplier(bundleId, containerType)
+    : containerRootSupplier;
   const resultPath = path.posix.resolve(containerRoot, match[2]);
```

Comment:
Maybe we could have a variable for `match[2]` as well, like you did with `bundleId`?

Reply:
added